src/Raven.Client/Documents/Operations/Replication/ReplicationHubAccess.cs

```diff
@@ -0,0 +1,263 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Http;
+using Raven.Client.Json;
+using Raven.Client.Json.Serialization;
+using Raven.Client.Util;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Client.Documents.Operations.Replication
+{
+    public class ReplicationHubAccess : IDynamicJson
+    {
+        public string Name;
+        public string CertificateBas64;
+        
+        public string[] Incoming;
+        public string[] Outgoing;
+
+        public DynamicJsonValue ToJson()
+        {
+            return new DynamicJsonValue
+            {
+                [nameof(Name)] = Name,
+                [nameof(Incoming)] = Incoming,
+                [nameof(Outgoing)] = Outgoing,
+                [nameof(CertificateBas64)] = CertificateBas64,
+                
+            };
+        }
+
+        public void Validate()
+        {
+            if (string.IsNullOrEmpty(Name))
+                throw new ArgumentNullException(nameof(Name));
+
+            if((Incoming?.Length ?? 0) == 0 && (Outgoing?.Length == 0))
```

Comment:
Can there be a scenario where I want block everything? (temporary?)

Reply:
Sure, set the allowed path to a guid, that would do it.